src/stores/spaces.ts

```diff
@@ -10,7 +12,8 @@ export const useSpacesStore = defineStore('spaces', {
     loadingMore: false,
     loaded: false,
     hasMoreSpaces: true,
-    spaces: [] as Space[]
+    spaces: [] as Space[],
+    spacesStarredIds: useStorage(`${pkg.name}.spaces-starred`, [] as string[])
```

Comment:
nit. I imagine it's pretty nit-picky, but `spacesStarred` seems weird (compared to `starredSpaces`).

Reply:
Changed to ``starredSpacesIds``, but I prefer to start with 'spaces' prefix for all spaces related vars, its better readability but maybe not perfect grammatically or weird :)
```
const state = {
  ...another state vars
  spaces: [],
  spacesSubscribedIds: [],
  spacesStarredIds,
  spaceSelected: null,
  ...another state vars
}
```
vs
```
const state = {
  ...another state vars
  spaces: [],
  subscribedSpacesIds: [],
  starredSpacesIds: [],
  selectedSpace: []
  ...another state vars
}
```
What do you think?

Reply:
although there all vars related to spaces in spaces store )